packaging/installer/methods/kickstart.md

```diff
@@ -5,181 +5,130 @@ import TabItem from '@theme/TabItem';
+| **Release Channel**     | `--release-channel`    | `nightly` or `stable`               |
+|                         | `--install-version`    | Install specific version            |
+| **Auto-Updates**        | `--auto-update`        | Enable updates                      |
+|                         | `--no-updates`         | Disable updates                     |
+| **Netdata Cloud**       | `--claim-token`        | Provide claim token                 |
+|                         | `--claim-rooms`        | Assign node to specific Cloud Rooms |
+| **Reinstall/Uninstall** | `--reinstall`          | Reinstall existing Netdata          |
+|                         | `--uninstall`          | Uninstall Netdata completely        |
 
-By default, the kickstart script will provide a Netdata Agent installation that can potentially communicate with Netdata Cloud if the Netdata Agent is further configured to do so.
+---
 
-- `--claim-token`
-  Specify a unique claiming token associated with your Space in Netdata Cloud to be used to connect to the node after the installation. This will connect and connect the Netdata Agent to Netdata Cloud.
-- `--claim-url`
-  Specify a URL to use when connecting to the Cloud. Defaults to `https://app.netdata.cloud`. Use this option to change the Netdata Cloud URL to point to your Netdata Cloud installation.
-- `--claim-rooms`
-  Specify a comma-separated list of tokens for each Room this node should appear in.
-- `--claim-proxy`
-  Specify a proxy to use when connecting to the Cloud in the form of `http://[user:pass@]host:ip` for an HTTP(S) proxy. See [connecting through a proxy](/src/claim/README.md#automatically-via-a-provisioning-system-or-the-command-line) for details.
-- `--claim-only`
-  If there is an existing installation, only try to connect it without attempting to update it. If there is no existing installation, install and connect Netdata normally.
+## Environment Variables
 
-### anonymous telemetry
+These environment variables provide additional customization options (most users won't need these):
 
-By default, the Agent is sending anonymous telemetry data to help us identify the most common operating systems and the configurations Netdata Agents run. We use this information to prioritize our efforts towards what is most commonly used by our community.
+| Variable            | Purpose                                      | Default Behavior                            |
+|---------------------|----------------------------------------------|---------------------------------------------|
+| `TMPDIR`            | Specify directory for temporary files        | System default temp directory               |
+| `ROOTCMD`           | Command to run with root privileges          | Uses `sudo`, `doas`, or `pkexec` (in order) |
+| `DISABLE_TELEMETRY` | Disable telemetry when set to non-zero value | Telemetry enabled                           |
 
-- `--disable-telemetry`
-  Disable anonymous statistics.
+> [!NOTE]
```

Comment:
Same comment as above, change the way of annotation so it renders on Docusaurus too